browser/ui/browser_commands.cc

```diff
@@ -285,4 +285,13 @@ bool HasSelectedURL(Browser* browser) {
   return brave_browser_window && brave_browser_window->HasSelectedURL();
 }
 
+void CleanAndCopySelectedURL(Browser* browser) {
+  if (!browser) {
+    return;
+  }
+  auto* brave_browser_window = BraveBrowserWindow::From(browser->window());
+  if (brave_browser_window) {
+    brave_browser_window->CleanAndCopySelectedURL();
+  }
```

Comment:
Curious this could be for `IDC_COPY_CLEAN_LINK` in `BraveBrowserCommandController::ExecuteBraveCommandWithDisposition`?
Honestly confusing when we can use `chrome::ExecuteCommand(browser, IDC_COPY_CLEAN_LINK);` or not.

Reply:
yes, here I removed that call